packages/fxa-settings/src/components/Settings/Nav/index.stories.tsx

```diff
@@ -38,7 +38,7 @@ const configWithoutNewsletterLink = {
 const storyWithContext = (
   account: Partial<Account>,
   storyName?: string,
-  config?: Config,
```

Comment:
It did. Hmm, do you mean [this line](https://github.com/mozilla/fxa/pull/14461/files#diff-e0f1c19f24d06ace0db76e995979d9beadc35fdc9d9104883c8c69183db83650L26)?

Reply:
Yeah, just because it was the file immediately above this. A bunch of the code above this uses trailing commas (like L17 in this same file), so was curious why Prettier only changed this one instance.

Minor nit, just curious if something funny is happening w/ Prettier, or if this was a manual tweak.

Reply:
On that line I just referenced, it's not a trailing comma but needed to separate arguments since it's using `storiesOf`. I'm probably misunderstanding though 😅

I did a find-and-replace for a couple of things, and manually modified/saved other files, so that might also be a factor.